zarr/tests/test_core.py

```diff
@@ -28,7 +28,17 @@
 )
 from numcodecs.abc import Codec
 from numcodecs.compat import ensure_bytes, ensure_ndarray
-from numcodecs.tests.common import greetings
+
+try:
+    from numcodecs.tests.common import greetings
+except ModuleNotFoundError:
+    greetings = [
+        '¡Hola mundo!', 'Hej Världen!', 'Servus Woid!', 'Hei maailma!',
+        'Xin chào thế giới', 'Njatjeta Botë!', 'Γεια σου κόσμε!',
+        'こんにちは世界', '世界，你好！', 'Helló, világ!', 'Zdravo svete!',
+        'เฮลโลเวิลด์'
+]
```

Comment:
I am unaware of the reason why `numcodecs.tests.common` cannot import this to the namespace – for this while, I have added this list of strings verbatim here. I will investigate the cause for the error.